src/styles/components/_text-input.css

```diff
@@ -0,0 +1,25 @@
+.input__text{
```

Comment:
Same thing about variations and properly separating blocks, elements and modifiers.